Sources/Fluent/Fluent+Response.swift

```diff
@@ -0,0 +1,106 @@
+import Vapor
+
+extension Response {
+    /// Creates a `Response` object that has a status of 201 (Created) and includes the `Location` HTTP header.
+    ///
+    /// Intended to be used in routes that create a new object.
+    ///
+    /// ### Note ###
+    /// The return type of your route should be `EventLoopFuture<Response>`.
+    ///
+    /// ### Example ###
+    /// ```swift
+    /// func create(req: Request) throws -> EventLoopFuture<Response> {
+    ///    let todo = try req.content.decode(Todo.self)
+    ///    return todo.create(on: req.db).flatMapThrowing {
+    ///        try .created(todo, for: req)
+    ///    }
+    /// }
+    /// ```
+    /// - Parameters:
+    ///   - obj: The newly created model.
+    ///   - request: The `Request` which is being responded to.
+    ///   - mediaType: How to encode the response, defaulting to `.json`
+    /// - Throws: If the object hasn't been created or the encoding fails.
+    /// - Returns: A `Response` object.
+    public static func created<T>(_ obj: T,
+                                  for request: Request,
```

Comment:
Xcode indentation.

Reply:
I don't know what you mean.  This is the way Xcode indents it.